src/frontend/efiling-frontend/src/components/page/rush/Rush.js

```diff
@@ -511,7 +546,9 @@ export default function Rush({ payment }) {
           label="The attached application is made under Rule 8-5 (1) SCR."
           onSelect={() => {
             setRadioStatusComponents();
+            setFields({ ...fields, rushType: "rule" });
             setRadio1(true);
+            resetFields();
```

Comment:
This should be re-ordered from:
setFields({ ...fields, rushType: "rule" });
setRadio1(true);
resetFields();

To:
resetFields();
setFields({ ...fields, rushType: "rule" });
setRadio1(true);

The reset is actually clearing the rushType. This applies to all three radio buttons.